index.js

```diff
@@ -1,20 +1,606 @@
-const readline = require('readline');
-const readlineInterface = readline.createInterface(process.stdin, process.stdout);
+//Boilerplate code set up correctly - used to accept input from user
+const { ENETRESET } = require("constants");
+const { read } = require("fs");
+const { exit } = require("process");
```

Comment:
Sometimes if VSCode auto corrects your code as something you weren't expecting, it can also auto import the dependencies. If these are things that aren't being used through your code, you can remove these lines to clean up some excess bloat